#goal definition
float64[] goToPositionxy
---
#result definition
int32 observation
---
#feedback
float64[] currentPosition
